services/kube-eleven/server/domain/utils/kube-eleven/kube_eleven.go

```diff
@@ -153,16 +155,67 @@ func (k *KubeEleven) generateTemplateData() templateData {
 
 	var potentialEndpointNode *spec.Node
 	data.Nodepools, potentialEndpointNode = k.getClusterNodes()
-
 	data.APIEndpoint = k.findAPIEndpoint(potentialEndpointNode)
 
+	hasHetznerNodes := k.hasHetznerNodes(data.Nodepools)
+	httpProxyMode := utils.GetEnvDefault("HTTP_PROXY_MODE", defaulHttpProxyMode)
+
+	data.NoProxy = ""
```

Comment:
You don't need to initialize with a default value, all values in Go are zero initialized, this line can be omitted.